src/translations/de.json

```diff
@@ -318,6 +318,8 @@
   "LayoutWrapperAccountSettingsSideNav.passwordTabTitle": "Passwort",
   "LayoutWrapperAccountSettingsSideNav.paymentsTabTitle": "Zahlungseinstellungen",
   "LayoutWrapperAccountSettingsSideNav.paymentMethodsTabTitle": "Zahlungseinstellungen",
+  "LimitedAccessBanner.message": "Your are logged in as {firstName} {lastName}. You have limited rights for actions.",
+  "LimitedAccessBanner.logout": "Log out",
```

Comment:
No need to add the English texts to other translation files. In case some key is missing from the translation file we will default to the English version in the code. (Same with es.json and dr.json)